src/__tests__/ReduxRouter-test.js

```diff
@@ -223,8 +223,6 @@ describe('<ReduxRouter>', () => {
       'or a class/function (for composite components) but got: object. ' +
       'Check the render method of `ReduxRouterContext`.';
 
-    const routingContextInvalidPropErrorMessage = 'Invalid prop `RoutingContext` of type `object` supplied to `ReduxRouterContext`';
```

Comment:
@Scarysize don't know why is that. if you redid `npm install` then maybe React got updated to 15.3
https://twitter.com/reactjs/status/759099486513704961